index.html

```diff
@@ -5,23 +5,46 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <title>Document</title>
+    <title>Awesome Books</title>
   </head>
 
   <body>
-    <h1>All Awesome Books</h1>
-    <div id="list"></div>
-    <form>
-      <hr />
+    <nav>
+      <h2>Awesome Books</h2>
+      <ul>
+        <li class="show-main">List</li>
+        <li class="add-new">Add New</li>
+        <li class="show-contact">Contact</li>
+      </ul>
```

Comment:
- [ ] OPTIONAL
To enhance the user experience of this application kindly ensure that an **active menu link is highlighted**(color change or change in opacity). In other words, the current active page should have the corresponding menu item highlighted, as can be seen in the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md). Kindly ensure to improve upon this aspect in your design, as we really want our designs to as much as possible mimic the provided wireframe template.

**Your nav menu:**
![image](https://user-images.githubusercontent.com/12745474/176731877-93f540b7-0cd4-47a8-b970-81ed8f749946.png)


**The wireframe nav menu:**
![image](https://user-images.githubusercontent.com/12745474/138342596-c4a68582-914a-4c5a-b255-8db0325cbaca.png)